core/commands/mount_darwin.go

```diff
@@ -6,7 +6,7 @@ import (
 	"strings"
 	"syscall"
 
-	fuseversion "github.com/jbenet/go-ipfs/Godeps/_workspace/src/github.com/jbenet/go-fuse-version"
+	fuseversion "github.com/jbenet/go-fuse-version"
```

Comment:
I'm all for anything that helps to reduce friction so long as it respects the developer's GOPATH.

I tend to use `godep restore ./...` to turn vendored dependencies into packages in my GOPATH, but I recognize that some may find this behavior inconvenient.